node-setup/Dockerfile

```diff
@@ -1,6 +1,9 @@
-FROM to-be-replaced-by-local-ref/base:ubuntu
+FROM to-be-replaced-by-local-ref/base:ubi-9.4-minimal
 
-RUN apt update && \
-    apt install -y --no-install-recommends xfsprogs mdadm && \
-    apt clean  && \
-    rm -rf /var/lib/apt/lists/*
+RUN microdnf update -y && \
+    curl -L https://repo.almalinux.org/almalinux/9/BaseOS/x86_64/os/RPM-GPG-KEY-AlmaLinux-9 -o /etc/pki/rpm-gpg/RPM-GPG-KEY-AlmaLinux && \ 
+    echo -e "[base-9]\nname=AlmaLinux 9 - \$basearch - Base\nbaseurl=https://repo.almalinux.org/almalinux/9/BaseOS/\$basearch/os/\ngpgcheck=1\nenabled=1\ngpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-AlmaLinux\n\n[appstream-9]\nname=AlmaLinux 9 - \$basearch - AppStream\nbaseurl=https://repo.almalinux.org/almalinux/9/AppStream/\$basearch/os/\ngpgcheck=1\nenabled=1\ngpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-AlmaLinux" > /etc/yum.repos.d/almaLinux.repo && \
```

Comment:
I think we should do it in a base image so we don't have to repeat this in the dependent images. There could be more than one. It can possibly be disabled by default if you want to be more explicit about choosing it and enabling it when you really want it on demand with `--enablerepo`.